Doc/library/dbm.rst

```diff
@@ -129,6 +134,69 @@ then prints out the contents of the database::
 
 The individual submodules are described in the following sections.
 
+.. Substitutions for the flag docs; all submodules use the same text.
+
+.. |flag_r| replace::
+   Open existing database for reading only (default)
+
+.. |flag_w| replace::
+   Open existing database for reading and writing
+
+.. |flag_c| replace::
+   Open database for reading and writing, creating it if it doesn't exist.
```

Comment:
All of this is duplicated for each dbm submodule. Instead of copying the text yet another time, set up replacements and use these instead. In a follow-up docs PR, we can reuse these for the other modules.

Reply:
Added in 8c265408c51609c6b4a6788cac9cc5fea7a14888.